CHANGELOG.md

```diff
@@ -244,27 +268,45 @@ Contributors:
 - [@monti-python](https://github.com/monti-python) ([#3247](https://github.com/fishtown-analytics/dbt/issues/3247))
 - [@drkarthi](https://github.com/drkarthi) ([#2426](https://github.com/fishtown-analytics/dbt/issues/2426), [#2998](https://github.com/fishtown-analytics/dbt/pull/2998))
 
-## dbt 0.19.2 (Release TBD)
+## dbt 0.19.2 (June 28, 2021)
+
+### Fixes
+
+- Fix infinite recursion when parsing schema tests due to loops in macro calls ([#3444](https://github.com/fishtown-analytics/dbt/issues/3344), [#3454](https://github.com/fishtown-analytics/dbt/pull/3454))
+
+## dbt 0.19.2rc2 (June 03, 2021)
 
 ### Breaking changes
+
 - Fix adapter.dispatch macro resolution when statically extracting macros. Introduce new project-level `dispatch` config. The `packages` argument to `dispatch` no longer supports macro calls; there is backwards compatibility for existing packages. The argument will no longer be supported in a future release, instead provide the `macro_namespace` argument. ([#3362](https://github.com/fishtown-analytics/dbt/issues/3362), [#3363](https://github.com/fishtown-analytics/dbt/pull/3363), [#3383](https://github.com/fishtown-analytics/dbt/pull/3383), [#3403](https://github.com/fishtown-analytics/dbt/pull/3403))
 
 ### Fixes
+
 - Fix references to macros with package names when rendering schema tests ([#3324](https://github.com/fishtown-analytics/dbt/issues/3324), [#3345](https://github.com/fishtown-analytics/dbt/pull/3345))
 
+## dbt 0.19.2rc1 (April 28, 2021)
+
+### Fixes
+
+- Ensure that schema test macros are properly processed ([#3229](https://github.com/fishtown-analytics/dbt/issues/3229), [#3272](https://github.com/fishtown-analytics/dbt/pull/3272))
+- Fix regression for default project/database for BigQuery connections ([#3218](https://github.com/fishtown-analytics/dbt/issues/3218), [#3305](https://github.com/fishtown-analytics/dbt/pull/3305))
```

Comment:
added as well

Reply:
Are we missing all of these b/c we made the commit in the release branch and not here?

Reply:
The fix was made in the wrong order at the time, it was added to 0.19.latest then cherry-picked to develop. I confirmed the fix is on develop (https://github.com/dbt-labs/dbt/commit/5956a64b0109f7b33681d37fa49364e900b730ac)